### PR TITLE
Remove one error and several respec warnings

### PR DESCRIPTION
**Pull request by plehegar**
Minor cleanup

***
[Preview](https://s3.amazonaws.com/pr-preview/w3c/hr-time/respec-errors-cleanup.html) | [Diff](https://s3.amazonaws.com/pr-preview/w3c/hr-time/f84f6ee...f8f20c7.html)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -301,20 +297,22 @@
...
-    'time-origin-timestamp'>timeOrigin</dfn> attribute MUST return a
+    <p>The <dfn data-dfn-for="Performance">timeOrigin</dfn> attribute MUST return a
     <a>DOMHighResTimeStamp</a> representing the high resolution time of the
     <a>time origin timestamp</a> for the [relevant global object] of the
     <a>Performance</a> object.</p>
+    <p>The <dfn data-dfn-for="Performance">serializer</dfn> provide a way for the <a>Performance</a>
+    object to be converted into a serialized form.</p>
   </section>
-  <section>
+  <section">
```
Comment (igrigorik) [inline index.html]: Typo?

File: index.html [hunk 2]
```diff
@@ -301,20 +297,22 @@
...
     <h3>The <code>performance</code> attribute</h3>
-    <p>The <a>GlobalPerformance.performance</a> attribute allows access to
-    performance related attributes and methods from the [global object].</p>
+    <p>The <dfn data-dfn-for="GlobalPerformance">performance</dfn> attribute allows access to
+    performance related attributes and methods from the [global object]. The <dfn>GlobalPerformance</dfn>
+    interface is intended to be exposed directly and will not exist in the ECMAScript binding.</p>
```
Comment (igrigorik) [inline index.html]: I'm unclear on this one. @annevk can you please sanity check?

Comment (annevk) [inline index.html]: I don't understand what this language means.

What you want to use here is a partial of https://html.spec.whatwg.org/#windoworworkerglobalscope-mixin and the language we use for attributes defined upon that.

Comment (plehegar) [inline index.html]: we missed that update (WindowOrWorkerGlobalScope wasn't there when we introduced GlobalPerformance). Aligned with the HTML spec.

**Comment by annevk** [comment]
The references to the W3C unmaintained fork of workers rather than the WHATWG HTML Standard will end up confusing implementers. I personally wouldn't accept that.

**Comment by plehegar** [comment]
The original goal of this PR was fix the respec errors,not to start rewriting the spec. I'd rather handle issues not related to respec fixing separetely.

**Comment by igrigorik** [comment]
Opened new issue for the reference: https://github.com/w3c/hr-time/issues/46. 

Merging this.

**Pull request merged**